shared/fs/really-delete/index.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+import * as React from 'react'
+import * as Constants from '../../constants/fs'
+import {Box, ConfirmModal, HeaderOnMobile, Icon, MaybePopup, ProgressIndicator} from '../../common-adapters'
+import {globalStyles, globalMargins, globalColors} from '../../styles'
+import * as Types from '../../constants/types/fs'
+
+export type Props = {
+  _deleting: boolean,
+  onBack: () => void,
+  onDelete: () => void,
+  path: Types.Path,
+  title: string,
+}
+
+const _Spinner = (props: Props) => (
+  <MaybePopup onClose={props.onBack}>
+    <Box
+      style={{...globalStyles.flexBoxColumn, alignItems: 'center', flex: 1, padding: globalMargins.xlarge}}
```

Comment:
Move these into `styles`?

Reply:
And prefer Kb.Box2 for new code instead of Kb.Box?